pages/index.js

```diff
@@ -1,50 +1,52 @@
 import Header from "@/components/Header/Header.js";
 import RoomList from "@/components/RoomList/RoomList.js";
 import { RoomType } from "@/components/RoomTile/RoomTile";
+import AddRoomModal from "@/components/AddRoomModal/AddRoomModal.js";
+import { useState } from "react";
```

Comment:
Looks good. Just small organisational aspect. We usually import external libraries first.

Reply:
Oh, thanks for the tip! I didn't know that